### PR TITLE
[10.0][MIG] mgmtsystem_nonconformity_project

### PR DESCRIPTION
**Pull request by thinkwelltwd**
#159 

This is the real version that I wanted. #190 was just to get moved up to v10. I know you'll complain that this PR wasn't following the [technical ](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0#technical-method-to-migrate-a-module-from-90-to-100-branch) and I didn't. 

But #190 isn't merged. And I tried branching off of my local copy of that branch to create v10, and that pulled in years of commits, like has been the bane of my contributions in [project](https://github.com/OCA/project). I tried a modified version of technical:

```
git format-patch --stdout upstream/10.0..origin/9.0-migrate-mgmsystem_nonconformity_project -- $MODULE | git am -3
```
And that generates no changes.  I would very much like to help migrate OCA modules, and I'd like to follow the workflow that the docs require. But I never have been able to create a v10 PR that actually was correct and only included my commits.

This is doubtless my fault, but just as certainly, it's an odd workflow to have 8,9,10 branches, all containing v8 code, and where v10 code is desired, yet v9 must be done first, yet where v9 and v10 have no continuity. 

I understand that it makes modules much more discoverable to have all branches include all modules. But it terms of workflow, well, let's say that it's beyond annoying.

I know you can't help everyone..but is there something you can do other than tell me to follow the directions, which I'm already well enough aware of? My email address is in the README.rst; I'd be happy to buy some coffee & donuts to someone who'll get me through this merge mess.

### CODE REVIEW

**Comment by dreispt** [comment]
The current migration strategy was designed to make simple the most common use case: migrating a module from the previous version to the current one, that had no later changes added to it.

In this case, you were able to start migrating a 8.0 module just by making it installable and making the needed changes, without any complicated git magic.
Not that the 9.0 PR is waiting on your feedback.

It's more efficient to have the 9.0 PR finished and merged before making the 10.0 - it will save you from repeating fixes. But nothing prevents you from working on the 10.0 at the same time.
The create the 10.0 port,  you can branch from 10.0 and the `git cherry-pick` the commits with your 9.0 changes.

File: mgmtsystem_nonconformity_project/README.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+Management-system/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+Management-system/issues/new?body=module:%20
+mgmtsystem_nonconformity_project%0Aversion:%20
+9.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (max3903) [inline mgmtsystem_nonconformity_project/README.rst:36]:
```suggestion
10.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

File: mgmtsystem_nonconformity_project/__manifest__.py [hunk 2]
```diff
@@ -1,43 +1,20 @@
...
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# -*- coding: utf-8 -*-
 {
     "name": "Management System - Project",
-    "version": "1.0",
+    "version": "10.0.1.0.0",
     "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
     "website": "http://www.savoirfairelinux.com",
```
Comment (max3903) [inline mgmtsystem_nonconformity_project/__manifest__.py:6]:
```suggestion
    "website": "https://github.com/OCA/management-system",
```

File: mgmtsystem_nonconformity_project/views/mgmtsystem_nonconformity_project.xml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<odoo>
+
+    <record id="view_mgmtsystem_action_tree" model="ir.ui.view">
+        <field name="name">mgmtsystem.action.tree</field>
+        <field name="model">mgmtsystem.action</field>
+        <field name="inherit_id" ref="mgmtsystem_action.view_mgmtsystem_action_tree"/>
+        <field name="arch" type="xml">
+
```
Comment (max3903) [inline mgmtsystem_nonconformity_project/views/mgmtsystem_nonconformity_project.xml:8]:
```suggestion
```

File: mgmtsystem_nonconformity_project/views/mgmtsystem_nonconformity_project.xml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            <field name="name" position="replace">
+                <field name="action_type"/>
+                <field name="complete_name"/>
+            </field>
+
```
Comment (max3903) [inline mgmtsystem_nonconformity_project/views/mgmtsystem_nonconformity_project.xml:13]:
```suggestion
```

File: mgmtsystem_nonconformity_project/views/mgmtsystem_nonconformity_project.xml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        </field>
+    </record>
+
+    <record id="view_mgmtsystem_action_form" model="ir.ui.view">
+        <field name="name">mgmtsystem.action.form</field>
+        <field name="model">mgmtsystem.action</field>
+        <field name="inherit_id" ref="mgmtsystem_action.view_mgmtsystem_action_form"/>
+        <field name="arch" type="xml">
+
```
Comment (max3903) [inline mgmtsystem_nonconformity_project/views/mgmtsystem_nonconformity_project.xml:22]:
```suggestion
```

File: mgmtsystem_nonconformity_project/views/mgmtsystem_nonconformity_project.xml [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+            <field name="name" position="replace">
+                <field name="action_type"/>
+            </field>
+
```
Comment (max3903) [inline mgmtsystem_nonconformity_project/views/mgmtsystem_nonconformity_project.xml:26]:
```suggestion
```

File: mgmtsystem_nonconformity_project/views/mgmtsystem_nonconformity_project.xml [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+            <field name="type_action" position="after">
+                   <field name="name" colspan="4"
+                          attrs="{'invisible' : [('action_type', '!=', 'action')]}"/>
+                   <field name="project_id" colspan="4"
+                          attrs="{'invisible' : [('action_type', '!=', 'project')]}"/>
+            </field>
+
```
Comment (max3903) [inline mgmtsystem_nonconformity_project/views/mgmtsystem_nonconformity_project.xml:33]:
```suggestion
```

**Pull request closed without merging**